ui-participant/src/participant/EditParticipantProfileModals.tsx

```diff
@@ -0,0 +1,331 @@
+import React, { useEffect, useState } from 'react'
+import { Profile } from '../api/api'
+import { javaLocalDateToJsDate, jsDateToJavaLocalDate, MailingAddress } from '@juniper/ui-core'
+import ThemedModal from '../components/ThemedModal'
+import Modal from 'react-bootstrap/Modal'
+import EditAddress from '@juniper/ui-core/build/components/EditAddress'
+
+/**
+ * Set of modals that are automatically picked based upon the `showFieldModal` prop,
+ * which denotes which field the user would like to edit.
+ */
+export default function EditParticipantProfileFieldModals(
```

Comment:
this component now doesn't add any functionality or state -- it's just a switch statement.  So let's turn it into an ordinary function and put it directly in ParticipantProfile.tsx